src/site/xdoc/server/config-cassandra.xml

```diff
@@ -319,6 +319,12 @@
 
     </subsection>
 
+    <subsection name="From V10 to V11">
+
+      <p>Add threadId column to messageIdTable and imapUidTable in order to get a message's threadId.</p>
```

Comment:
We should give the commands on Cassandra to perform for that I believe. But I consider that you probably don't know them.

I will try to support you on this part when I have a bit of time.

I think we need to modify the proper section in the new doc with Antora too :)